doc/man3/DTLS_set_timer_cb.pod

```diff
@@ -0,0 +1,36 @@
+=pod
+
+=head1 NAME
+
+DTLS_set_timer_cb - Set callback for controlling DTLS timer duration
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ void DTLS_set_timer_cb(SSL *s, unsigned (*cb)(SSL *s, unsigned timer));
```

Comment:
This should be updated with the same "unsigned int" change as is used later. Actually you could also include the typedef in the man page as is done on a few of the other man pages.